src/features/tags/components/TagsList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Tag } from "../types";
+import TagItem from "./TagItem";
+
+type TagsListProps = {
+  tags: Tag[] | undefined;
+  isHidden: string;
+};
+
+const TagsList: React.FC<TagsListProps> = ({ tags, isHidden }) => {
+  let tagsList;
+  if (tags) {
+    tagsList = tags.map((tag) => {
```

Comment:
tagsList is looping over the array of tag ids, so there tag parameter here is really just a single id that needs to be passed to TagItem

Reply:
This helps a lot thanks!